components/fraudlabs_pro/actions/verify-otp/verify-otp.mjs

```diff
@@ -0,0 +1,59 @@
+import  * as flpnodejs from 'fraudlabspro-nodejs';
```

Comment:
If you modify the file `fraudlabs_pro.app.mjs` as per my suggestion. then you can replace this import with
```suggestion
import fraudlabsProApp from "../../fraudlabs_pro.app.mjs";
```